index.html

```diff
@@ -4715,12 +4715,21 @@ <h2>Verification Method Revocation</h2>
 <br/><br/>
 For example, for a DID method that supports the `versionId` query parameter as
 well as the `updated` and `nextUpdate` DID document metadata properties, a
-verifier can validate a signature or proof of a revoked key iif: (1) the proof
-or signature contains the `versionId` to refer to the DID document that was
-used at the point the signature or proof was created. (2) The verifier can
-determine at what point in time the signature or proof was made (e.g. it was
-anchored on a blockchain). (3) For the DID document resolved using `versionId`,
-the `updated` timestamp is before 2 and the `nextUpdate` timestamp is after 2.
+verifier can validate a signature or proof of a revoked key IFF:
```

Comment:
```suggestion
verifier can validate a signature or proof of a revoked key if and only if 
all of the following are true:
```
because we can't be sure that `IFF` will be understood

Reply:
Fair enough 👍